### DISCUSSION TITLE
Is there a way to reload theme files during development?

### DISCUSSION THREAD

**Discussion by atkulp** [GetPublii/Publii#1728] [General]
I keep closing and re-opening the main app (Windows 11), but it would be much nicer to be able to click somewhere to just reload my theme.

**Comment by atkulp** [comment]
Never mind! I was updating the wrong thing so the change wasn't showing.

**Comment by Julianoe** [comment]
@atkulp how do you do it?
I'm starting to try to dev a custom theme for Publii (no blog, 3 pages, local shop type), but I have the same issue that you describe, always having to close and reopen the main app to reload the theme. How do you do it?

**Reply by bobmitro** [reply]
Do not edit or develop theme files directly inside the main theme folder.
Instead, do it via your site directory: Documents ▸ Publii ▸ sites ▸ YOUR_SITE ▸ input ▸ themes ▸ YOUR_THEME

That way, you’ll be able to preview your site using the Preview button in the Publii app.

In general, the best workflow for development is first to create a static HTML version of your site, and then convert (dress) it into Handlebars.
